cg/store/models.py

```diff
@@ -680,15 +671,16 @@ class Flowcell(Base):
     archived_at: Mapped[datetime | None]
     has_backup: Mapped[bool] = mapped_column(default=False)
     updated_at: Mapped[datetime | None] = mapped_column(onupdate=datetime.now)
-
-    samples: Mapped[list["Sample"]] = orm.relationship(
-        secondary=flowcell_sample, back_populates="flowcells"
-    )
     sequencing_metrics: Mapped[list["SampleLaneSequencingMetrics"]] = orm.relationship(
         back_populates="flowcell",
         cascade="all, delete, delete-orphan",
     )
 
+    @property
+    def samples(self) -> list["Sample"]:
+        """Return samples sequenced on the flow cell."""
+        return list({metric.sample for metric in self.sequencing_metrics})
+
```

Comment:
Yes, list is to not change any functionality. Maybe we could do set now that I think about it 🤔 Set is to remove duplicates since the same sample will have one metric per lane.

Reply:
maybe out of scope? Feel free to resolve for now though

Reply:
Keeping as list for now